docs/.vitepress/config.mjs

```diff
@@ -303,49 +130,10 @@ export default defineConfig({
       pattern: "https://github.com/tuist/tuist/edit/main/docs/docs/:path",
     },
     sidebar: {
-      "/contributors": [
-        {
-          text: "Contributors",
-          items: [
-            {
-              text: "Get started",
-              link: "/contributors/get-started",
-            },
-            {
-              text: "Issue reporting",
-              link: "/contributors/issue-reporting",
-            },
-            {
-              text: "Code reviews",
-              link: "/contributors/code-reviews",
-            },
-            {
-              text: "Principles",
-              link: "/contributors/principles",
-            },
-          ],
-        },
-      ],
-      "/guide/": guideSidebar,
-      "/": guideSidebar,
-      "/reference/": [
-        {
-          text: "Reference",
-          items: [
-            cliSidebar,
-            projectDescriptionSidebar,
-            {
-              text: "Examples",
-              items: examplesDataLoader.load().map((item) => {
-                return {
-                  text: item.title,
-                  link: `/reference/examples/${item.name}`,
-                };
-              }),
-            },
-          ],
-        },
-      ],
+      "/contributors": contributorsSidebar,
+      "/guides/": guidesSidebar,
+      "/": guidesSidebar,
+      "/references/": referencesSidebar,
```

Comment:
What do you think of adding `Sign in` and `Sign up` button in the top bar as fly.io does?
<img width="1510" alt="image" src="https://github.com/user-attachments/assets/23acde37-338a-453a-bdd5-90715271d8e4">

Reply:
I like the idea! I'll add a link.